src/sources/audiosourcetrackproxy.h

```diff
@@ -1,20 +1,18 @@
 #ifndef MIXXX_AUDIOSOURCETRACKPROXY_H
 #define MIXXX_AUDIOSOURCETRACKPROXY_H
 
-
 #include "sources/audiosource.h"
 
 #include "track/track.h"
 
-
```

Comment:
Is this a clang-format  change?

Reply:
Yes. I don't care about blank lines as long as it is consistent and readable.